.ci-config/rippled.cfg

```diff
@@ -0,0 +1,151 @@
+[server]
+port_rpc_admin_local
+port_ws_public
+port_ws_admin_local
+
+# port_peer
+# port_ws_admin_local
+# ssl_key = /etc/ssl/private/server.key
+# ssl_cert = /etc/ssl/certs/server.crt
+
+# IPs must be 0.0.0.0 instead of 127.0.0.1 to be accessed outside the docker container
+
+[port_rpc_admin_local]
+port = 5005
+ip = 0.0.0.0
+admin = 0.0.0.0
+protocol = http
+
+[port_ws_public]
+port = 80
+ip = 0.0.0.0
+protocol = ws
+
+# [port_peer]
+# port = 51235
+# ip = 0.0.0.0
+# protocol = peer
+
+[port_ws_admin_local]
+port = 6006
+ip = 0.0.0.0
+admin = 0.0.0.0
+protocol = ws
+
+[node_size]
+small
+
+# tiny
+# small
+# medium
+# large
+# huge
+
+[node_db]
+type=NuDB
+path=/var/lib/rippled/db/nudb
+advisory_delete=0
+
+# How many ledgers do we want to keep (history)?
+# Integer value that defines the number of ledgers
+# between online deletion events
+online_delete=256
+
+[ledger_history]
+# How many ledgers do we want to keep (history)?
+# Integer value (ledger count)
+# or (if you have lots of TB SSD storage): 'full'
+256
+
+[database_path]
+/var/lib/rippled/db
+
+[debug_logfile]
+/var/log/rippled/debug.log
+
+[sntp_servers]
+time.windows.com
+time.apple.com
+time.nist.gov
+pool.ntp.org
+
+[ips]
+r.ripple.com 51235
+
+[validators_file]
+validators.txt
+
+[rpc_startup]
+{ "command": "log_level", "severity": "info" }
+
+# severity (order: lots of information .. only errors)
+# debug
+# info
+# warn
+# error
+# fatal
+
+[ssl_verify]
+1
+
+# The [features] stanza does not currently work for standalone mode: https://github.com/XRPLF/xrpl-dev-portal/issues/1762#issuecomment-1441252450
+
+
+# In order to enable an amendment which by default would vote "No", you must include its amendment id and name here.
+# To get the list of amendments on a network:
+# 1. Run this ledger_entry command against the network to get a list of enabled amendment ids. (Command is in the websocket link as an easy way to run it)
+#    https://xrpl.org/websocket-api-tool.html?server=wss%3A%2F%2Fs1.ripple.com%2F&req=%7B%22command%22%3A%22ledger_entry%22%2C%22index%22%3A%227DB0788C020F02780A673DC74757F23823FA3014C1866E72CC4CD8B226CD6EF4%22%2C%22ledger_index%22%3A%22validated%22%7D
+# 2. Strip away the quotes and commas
+# 3. Add the amendment name to the same line as each amendment id (You can look them up via hash on https://xrpl.org/known-amendments.html)
+#   Ex. 4C97EBA926031A7CF7D7B36FDE3ED66DDA5421192D63DE53FFB46E43B9DC8373 Multisign
+#   The amendment name can be any string (including just a number)
+#
+# Note: The version of rippled you use this config with must have an implementation for the amendments you attempt to enable or it will crash.
+
+[amendments]
+# Devnet amendments as of March 28th, 2023
+B4E4F5D2D6FB84DF7399960A732309C9FD530EAE5941838160042833625A6076 NegativeUNL
```

Comment:
Is there a way to avoid making this manual? Perhaps we can inject it via CI/CD at build time by downloading a list from the rippled repo?

Reply:
I think it would have to look at the Devnet network to do that because the Rippled repo doesn't have information on which amendments are actually voted in. I think automating it is definitely doable. I have a separate task in our backlog though to automate it because I think it's a more involved ask. (For now I think this PR is a win because it at least gets us back up to date & not stuck on an old version which we can't update even manually)

Reply:
Can you link the github issue for the followup work here?

Reply:
https://github.com/XRPLF/xrpl-py/issues/578